examples/specs/layer_ranged_dot.vl.json

```diff
@@ -0,0 +1,85 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A ranged dot plot that uses 'layer' to convey changing life expectancy for the five most populous countries (between 1955 and 2000).",
+  "data": {"url": "data/countries.json"},
+  "transform": [
+      {
+          "filter": {
+              "field": "country",
+              "oneOf": ["China", "India"
+              , "United States", "Indonesia", "Brazil"]
+          }
+      },
+      {
+          "filter": {
+              "field": "year",
+              "oneOf": [1955, 2000]
+          }
+      }
+  ],
+  "layer": [
+      {
+          "mark": "point",
+          "encoding": {
+              "x": {
+                  "field": "life_expect",
+                  "type": "quantitative",
+                  "axis": {"title": "Life Expectancy (years)"}
+              },
+              "y": {
+                  "field": "country",
+                  "type": "nominal",
+                  "axis": {
+                      "title": "Country",
+                      "offset": 5,
+                      "ticks": false,
+                      "minExtent": 70,
+                      "domain": false
+                  }
+              },
+              "color": {
+                  "field": "year",
+                  "type": "ordinal",
+                  "scale": {
+                      "domain": [1955, 2000],
+                      "range": ["#e6959c",
+                      "#911a24"]
+                  },
+                  "legend": {
+                      "title": "Year"
+                  }
+              }
+          }
+      },
+      {
+          "mark": "line",
+          "encoding": {
+              "x": {
+                  "field": "life_expect",
+                  "type": "quantitative"
+              },
+              "y": {
+                  "field": "country",
+                  "type": "nominal"
+              },
+              "opacity": {
```

Comment:
Not sure why you are mapping a field here (while setting range to be constant). 

I think you actually wanna use `{value: 0.2}` ?

Reply:
I used 'field' as a workaround to group points by 'country' such that the lines would be drawn between points of the same country.

Using just 'value', I get the following:
```              
"opacity": {
    "value": 0.2
}
```
<img width="386" alt="screen shot 2017-10-06 at 11 16 45 am" src="https://user-images.githubusercontent.com/11954298/31292406-1b9421fe-aa88-11e7-8684-97db156ded8b.png">

Maybe there is a different workaround?

Reply:
ah I see, the right way to separate line would be using `detail` channel. 

However, a better alternative would be using `rule` (line segment) instead .